main.go

```diff
@@ -63,6 +63,7 @@ var (
 	metricsPort             = flag.String("metrics-port", "10254", "Port for the metrics endpoint.")
 	metricsEndpoint         = flag.String("metrics-endpoint", "/metrics", "Metrics endpoint.")
 	metricsPrefix           = flag.String("metrics-prefix", "", "Prefix for the metrics.")
+	defaultSparkImage       = flag.String("default-spark-image", "", "Default unified container image for the Spark application (for both the driver and executor)")
```

Comment:
I think it should have a default value.

Reply:
What do you suggest?

Reply:
I think we can use the same default image as the Helm Chart.